oio/api/object_storage.py

```diff
@@ -89,12 +91,15 @@ def __init__(self, namespace, logger=None, perfdata=None, **kwargs):
         conf = {"namespace": self.namespace}
         self.logger = logger or get_logger(conf)
         self.perfdata = perfdata
+        self.cache = cache
```

Comment:
No need for this field. Also, I don't think `cache` needs to appear in this method's signature. Adding `'cache'` to `EXTRA_KEYWORDS` would do the trick.